Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -43,6 +47,8 @@ enum SwiftPackageManagerGraphGeneratorError: FatalError, Equatable {
             return "The \(platform) platform is not supported."
         case let .unknownProductDependency(name, package):
             return "The product \(name) of the package \(package) cannot be found."
+        case let .unsupportedDependencyKind(name):
+            return "The dependency kind \(name) is not supported."
```

Comment:
The more information we add to errors, and the more actionable we make them, the better.
```
The package dependency 'MyPackage' is of type 'xxx', which is not yet supported by Tuist.
```
Because that's not actionable enough, I'd extend the [logic](https://github.com/tuist/tuist/blob/main/Sources/TuistSupport/Errors/ErrorHandler.swift#L30) that handles the errors, to print the following in case of `abort` errors:

```
Consider creating an issue using the following link: https://github.com/tuist/tuist/issues/new/choose
```